packages/flutter/test/widgets/semantics_tester.dart

```diff
@@ -505,8 +519,9 @@ class TestSemantics {
     if (id != null) {
       buf.writeln('$indent  id: $id,');
     }
-    if (flags is int && flags != 0 ||
-        flags is List<SemanticsFlag> && (flags as List<SemanticsFlag>).isNotEmpty) {
+    if ((flags is int && flags != 0) ||
```

Comment:
Do we have existing API to convert `int`, `List<SemanticsFlag>`  into SemanticsFlags? if so consider store the class property in SemanticsFlags and do the conversion in constructor

Reply:
Do we have existing API to convert int, List<SemanticsFlag> into SemanticsFlags?  >> we only have existing API to convert SemanticsFlags to int, but not int to SemanticsFlags.